crds/backupdr_v1alpha1_backupdrbackupvault.yaml

```diff
@@ -221,6 +217,10 @@ spec:
                     description: Output only. Set to true when there are no backups
                       nested under this resource.
                     type: boolean
+                  etag:
```

Comment:
@justinsb Scope of etag changed in the v1alpha1. As per yesterdays chat, wondering if this should be converted to v1alpha2 based on the change being not backward compatible.